deploy/cr.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   sslInternalSecretName: my-cluster-ssl-internal
   allowUnsafeConfigurations: false
 #  pause: false
-  updateStrategy: SmartUpdate
+#  updateStrategy: RollingUpdate
```

Comment:
please revert default

Reply:
But this is default

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/408/commits/29335ef3ef25187e4bdfcebcccd1774c6748678c